packages/preview/quill-0.1.0/README.md

```diff
@@ -0,0 +1,58 @@
+<h1 align="center">
+  <img alt="Quantum Circuit" src="https://github.com/Mc-Zen/packages/assets/52877387/5d34c646-79a8-492b-8e49-9136d5881258" style="max-width: 100%; width: 300pt; padding: 10px 20px; box-shadow: 1pt 1pt 10pt 0pt #AAAAAA; border-radius: 4pt;box-sizing: border-box;">
+</h1>
+
+<!-- <p align="center">
+  <a href="https://github.com/Mc-Zen/quill/blob/main/LICENSE">
+    <img alt="MIT License" src="https://img.shields.io/badge/license-MIT-blue"/>
+  </a>
+</p> -->
+
+**Quill** is a package for
+ creating quantum circuit diagrams in [Typst](https://typst.app/). 
+
+
+_Note, that this package is in beta and may still be undergoing breaking changes. As new features like data types and scoped functions will be added to Typst, this package will be adapted to profit from the new paradigms._
+
+_Meanwhile, we suggest importing everything from `quill.typ` in a local scope to avoid polluting the global namespace (see example below)._
+
+<!-- ## Setup
+
+Since there is no package manager for Typst yet, in order to use this library, download the [quill.typ](./qcircuit.typ) file and place it in your Typst project.  -->
+
+## Usage
+
+Create circuit diagrams by calling the function `quantum-circuit()` with any number of positional arguments — just like the built-in Typst functions `table()` or `grid()`. A variety of different gate and instructions commands are available and plain numbers can be used to produce any number of empty cells just filled with the current wire mode. A new wire is started by adding the content item `[\ ]`. 
+
+```java
+#{
+  import "quill.typ" : *
```

Comment:
this import should show how to import as a package, i.e. `import "@preview/quill:0.1.0": *`

Reply:
Yes, I'll change that!